homeassistant/components/vallox/__init__.py

```diff
@@ -133,10 +136,30 @@ class ValloxDataUpdateCoordinator(DataUpdateCoordinator):
 
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the integration from configuration.yaml (DEPRECATED)."""
+    if DOMAIN not in config:
+        return True
+
+    _LOGGER.warning(
+        "Loading vallox via platform config is deprecated. The configuration has been migrated"
+        " to a config entry and can be safely removed from configuration.yaml"
+    )
+
```

Comment:
This isn't setup as a platform.  Use `cv.deprecated(DOMAIN)` instead (example in `heos`)

Reply:
Ah, ok 👍 

Like this then?

```python
CONFIG_SCHEMA = vol.Schema(
    vol.All(
        cv.deprecated(DOMAIN),
        {
            DOMAIN: vol.Schema(
                {
                    vol.Required(CONF_HOST): vol.All(ipaddress.ip_address, cv.string),
                    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
                }
            )
        },
    ),
    extra=vol.ALLOW_EXTRA,
)
```